app/package.json

```diff
@@ -4,16 +4,14 @@
   "description": "Oran.js JavaScript by Orange Cats",
```

Comment:
This change is causing the Travis-CI build to fail. `package.json` needs to stay in the root and really I'd like to see as much of the app stay in the root as possible. Since this is a fun project with a lot of PRs, this is going to conflict with a lot of people who have forked and are working on PRs now.